docs/user/who-is.rst

```diff
@@ -0,0 +1,62 @@
+WHOIS Lookup
+============
+
+.. important::
+
+    The WhoIs Lookup feature is disabled by default.
```

Comment:
```suggestion
    The **WHOIS Lookup** feature is disabled by default.
```

Let's be consistent with the naming, are you ok with `WHOIS Lookup`? If yes, please make the wording consistent across all files.